xchange-kraken/src/main/java/org/knowm/xchange/kraken/service/KrakenAccountService.java

```diff
@@ -92,50 +90,108 @@ public String withdrawFunds(WithdrawFundsParams params) throws IOException {
     throw new IllegalStateException("Don't know how to withdraw: " + params);
   }
 
-  @Override
-  public String requestDepositAddress(Currency currency, String... args) throws IOException {
-    KrakenDepositAddress[] depositAddresses;
-    if (Currency.BTC.equals(currency)) {
-      depositAddresses = getDepositAddresses(currency.toString(), "Bitcoin", false);
+  private String getMethod(Currency currency, String protocol) throws IOException {
+    KrakenDepositMethods[] depositMethods = getDepositMethods(null, currency.toString());
+
+    if(depositMethods == null || depositMethods.length <= 0) {
+      return null;
+    }
+
+    if(depositMethods.length > 1 && protocol == null) {
+      LOGGER.warn("Multiple methods for currency "+currency+" protocol "+protocol);
+      for(KrakenDepositMethods method : depositMethods) {
+        LOGGER.warn(" -- "+method.getMethod());
+      }
```

Comment:
change the above to `LOGGER.warn("Multiple methods for currency {} protocol {}, returning first method from: {}", currency, protocol, depositMethods);`